lib/workers/types.ts

```diff
@@ -69,6 +69,7 @@ export interface BranchUpgradeConfig
   changelogUrl?: string;
   dependencyUrl?: string;
   sourceUrl?: string;
+  sourceRepo?: string;
```

Comment:
```suggestion
  sourceRepo?: string;
  sourceRepoOrg?: string;
  sourceRepoName?: string;
```

We should add all new fields 😉